doc/doxygen_project.txt.in

```diff
@@ -72,7 +72,7 @@ WARN_LOGFILE           =
 #---------------------------------------------------------------------------
 # configuration options related to the input files
 #---------------------------------------------------------------------------
-INPUT                  = $MRPT_LIST_INPUT
+INPUT                  = $MRPT_LIST_INPUT "../libs/vision/src/pnp"

```

Comment:
ok I see now! This is to include `src/*` files, not `include/*` files! 
Then, please better replace this addition here with a new file [here](https://github.com/MRPT/mrpt/blob/master/scripts/build_docs.sh#L69) and its concatenation [here](https://github.com/MRPT/mrpt/blob/master/scripts/build_docs.sh#L79), for consistency with the rest of "extra/especial" doxygen files.